### PR TITLE
Corporate Action Announcements Documentation

### PR DESCRIPTION
**Pull request by TamirHFrank**

### CODE REVIEW

**Review by alpacadave — APPROVED** [review]
looks good to me

File: content/api-references/corporate-actions.md [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+| Attribute          | Description                                                                                                         |
+| -----------------  | ------------------------------------------------------------------------------------------------------------------- |
+| `declaration_date` | The date of the preliminary announcement details or the date that any subsequent term updates took place.           |
+| `ex_date`          | The first date that any purchases of a security will not result in a corporate action entitlement.                  |
+| `record_date`      | The date that an account must hold a position in the security in order to receive the corporate action entitlement. |
+| `payable_date`     | The date that the stock and cash positions will update according to the account positions as of the record date.    |
+
+---
+
+## **Retrieving Corporate Action Announcements**
```
Comment (umitanuki) [inline content/api-references/corporate-actions.md:62]: Does it have to be bold?

File: content/api-references/corporate-actions.md [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+​ _Invalid input value._
+{{</hint>}}
+
+{{<hint warning>}}
+500 - Internal Server Error​
+
+_Some server error occurred. Please contact Alpaca._
+{{</hint>}}
+
+&nbsp;
```
Comment (umitanuki) [inline content/api-references/corporate-actions.md:194]: What's this?

File: content/api-references/corporate-actions.md [hunk 3]
```diff
@@ -0,0 +1,194 @@
...
+{{<hint warning>}}
+400 - Bad Request
+
+​ _The body in the request is not valid_
+{{</hint>}}
+
+{{<hint warning>}}
+404 - Bad Request
+
+​ _Relationship not found_
```
Comment (umitanuki) [inline content/api-references/corporate-actions.md:179]: What relationship?

**Pull request closed without merging**